### PR TITLE
Feature/admin overhaul

### PR DESCRIPTION
**Pull request by hixi**
Adds support for listing the user's exports and its dependent data in the admin.

### CODE REVIEW

File: osmaxx/excerptexport/admin.py [hunk 1]
```diff
@@ -35,6 +53,36 @@ class OutputFileAdmin(admin.ModelAdmin):
...
+    readonly_fields = ('finished_at', 'created_at', 'updated_at', 'link_to_extraction_order')
+
+    def link_to_extraction_order(self, obj):
+        link = urlresolvers.reverse("admin:excerptexport_extractionorder_change", args=[obj.extraction_order.id])
+        return u'<a href="%s">%s</a>' % (link, obj.extraction_order.excerpt_name)
+    link_to_extraction_order.allow_tags = True
+
+
+User = get_user_model()
+admin.site.unregister(User)
```
Comment (das-g) [inline osmaxx/excerptexport/admin.py]: I'd put these two lines directly above
```python
@admin.register(User)
class UserAdmin2(UserAdmin):
    # ...
```
below.

File: osmaxx/excerptexport/admin.py [hunk 2]
```diff
@@ -35,6 +53,36 @@ class OutputFileAdmin(admin.ModelAdmin):
...
+    show_change_link = True
+    extra = 0
+
+    def has_add_permission(self, request):
+        return False
+
+
+@admin.register(User)
+class UserAdmin2(UserAdmin):
+    inlines = [
```
Comment (das-g) [inline osmaxx/excerptexport/admin.py]: As `UserAdmin` might declare its own `inlines` in the future, should this be
```python
    inlines += [
        # ...
```
or would we _want_ to override pre-existing `inlines` in that case?

File: osmaxx/excerptexport/admin.py (hunk 1 above)
Comment (hixi) [inline osmaxx/excerptexport/admin.py]: good idea, I missed that one.

File: osmaxx/excerptexport/admin.py (hunk 2 above)
Comment (hixi) [inline osmaxx/excerptexport/admin.py]: wouldn't this result in "unresolved reference"?

**Pull request merged**